src/Psl/Result/IResult.php

```diff
@@ -11,12 +11,12 @@
  * Represents a result of operation that either has a successful result or the exception object if
  * that operation failed.
  *
- * This is an interface. You get generally `IResultOrExceptionWrapper<T>` by calling `wrap<T>()`, passing in
- * the `callable(): T`, and a `WrappedResult<T>` or `WrappedException<Te>` is returned.
+ * This is an interface. You get generally `IResult<T>` by calling `tryResultFrom<T>()`, passing in
+ * the `callable(): T`, and a `Ok<T>` or `Failure<Te>` is returned.
  *
  * @template T
  */
-interface IResultOrExceptionWrapper
+interface IResult
```

Comment:
let's renaming to ResultInterface instead, previously PSL used `ISomething` naming instead of `SomethingInterface` ( similar to C#, and Hack ), but I think `SomethingInterface` is more commonly used in the PHP ecosystem.

Reply:
Yeah, is better!